synthtool/gcp/gapic_microgenerator.py

```diff
@@ -183,7 +183,7 @@ def _generate_code(
                 docker_run_args.append(value)
 
         logger.debug(f"Generating code for: {proto_path}.")
-        shell.run(docker_run_args)
+        shell.run(docker_run_args, hide_output=False)
```

Comment:
Are any secrets, like github tokens, passed to docker via command line arguments and would potentially appear in the logs?

Reply:
This is not a concern for Python (there are no secrets in the generator image and we don't pass any secrets in as args)

Reply:
Same with Ruby. The generator itself shouldn't be dealing with any secrets, github or otherwise.